cmd/readPipelineEnv.go

```diff
@@ -37,6 +44,20 @@ func runReadPipelineEnv() error {
 		return err
 	}
 
+	// try to encrypt
+	if secret, ok := os.LookupEnv("PIPER_pipelineEnv_SECRET"); ok && secret != "" {
```

Comment:
Where the PIPER_pipeline Env_SECRET does come from? Where it can be set up?

if I remember correctly, we discussed that the step(s) (read/writePipelineEnv) should have the possibility to fetch token/secret/key from Vault.
Is there a plan to implement it?

Reply:
I think it is a better way to do it inside the go step, but we need to use a workaround, because these two steps are internal, and they are not generated from yaml step specs